python/lsst/meas/extensions/piff/piffPsfDeterminer.py

```diff
@@ -72,10 +72,27 @@ def _validateGalsimInterpolant(name: str) -> bool:
 
 
 class PiffPsfDeterminerConfig(BasePsfDeterminerTask.ConfigClass):
+    spatialOrderPerBand = pexConfig.DictField(
+        doc="Per-band spatial order for PSF kernel creation. "
+        "Ignored if piffPsfConfigYaml is set.",
+        keytype=str,
+        itemtype=int,
+        default={
```

Comment:
I would be for letting a fallback value here = 2 to let the default behavior to be the same as before.

Reply:
Wait, that's what it is...are you just agreeing, or do you think something needs changing?